jscripts/bbcodes_sceditor.js

```diff
@@ -480,12 +480,12 @@ $(document).ready(function($) {
 						url     = matches ? 'https://www.facebook.com/video/embed?video_id=' + matches[1] : false;
 						break;
 					case 'liveleak':
-						matches = content.match(/liveleak\.com\/(?:view\?i=)([^\/]+)/);
+						matches = content.match(/liveleak\.com\/(?:view\?[a-z]{1}=)([^\/]+)/);
```

Comment:
This `{1}` shouldn't be necessary here surely? Shouldn't `[a-z]` alone achieve the same thing, or am I missing something?

Reply:
Of course! My bad.